src/browser/wpe/opencdm/open_cdm.cpp

```diff
@@ -161,38 +157,32 @@ int OpenCdm::Load(std::string& responseMsg) {
   return ret;
 }
 
-int OpenCdm::Update(unsigned char* pbResponse, int cbResponse, std::string& responseMsg) {
-
-  CDM_DLOG() << "Update >> invoked from ocdm :: estate = " << m_eState << "\n";
+int OpenCdm::Update(unsigned char* pbResponse, int cbResponse, std::string& responseMsg)
+{
```

Comment:
This code seems to have incoherent style.

Reply:
Yes, yes it does :)